src/app/plan/components/Wizard.tsx

```diff
@@ -86,11 +95,11 @@ class WrappedWizard extends React.Component<any, any> {
             handleChange={handleChange}
             setFieldValue={setFieldValue}
             setFieldTouched={setFieldTouched}
-            clusterList={clusterList}
+            onWizardLoadingToggle={this.handleWizardLoadingToggle}
           />
         ),
         // enableNext: !errors.planName && touched.planName === true
-        enableNext: !errors.sourceCluster && touched.sourceCluster === true,
+        enableNext: !errors.sourceCluster && touched.sourceCluster === true && !this.state.isWizardLoading,
```

Comment:
need a better way of handling next state enabled. Maybe redux for this also.

Reply:
Yeah, this was exactly the issue I had with being able to disable next in the modal. The state is not easily reachable; I think the modal's state should be migrated to the redux tree.